src/Servers/Kestrel/shared/test/TestConstants.cs

```diff
@@ -8,6 +8,6 @@ namespace Microsoft.AspNetCore.Testing
     public class TestConstants
     {
         public const int EOF = -4095;
-        public static TimeSpan DefaultTimeout = TimeSpan.FromSeconds(30);
+        public static TimeSpan DefaultTimeout = TimeSpan.FromMinutes(1);
```

Comment:
@JamesNK @Tratcher @halter73 @BrennanConroy this seemed painless to me and is definitely the simplest option. But please let me know if you object to such a general change for what's mostly a single test failing.

Reply:
My preference is to increase the timeout on the individual test. But if that isn't possible then I don't have a problem with this.

Perhaps add a comment to say what test the 1 minute timeout is for?

Reply:
I was just saving myself some copying and pasting from `TaskTimeoutExtensions` and `TestConstants`. Given your preference, I'll switch to the focused approach after dinner or tomorrow morning.